SideBar.py

```diff
@@ -17,6 +17,24 @@ def get_path(self, paths):
 		except IndexError:
 			return self.window.active_view().file_name()
 
+	@staticmethod
+	def make_dirs_for(filename):
+		"""
+		Attempts to create all necessary subdirectories for `filename`.
+
+		Returns True if the tree was created, False otherwise, but doesn't
+		mean we can or cannot write to it. It simply means it was not created
+		by this function.
+		"""
```

Comment:
I have fairly aggressive linter settings, it wants docstrings on everything. As a consequence of spending too much time dealing with large codebase, documentation is important! But fair enough, this is a simple plugin, and we want to keep it that way.